harness/determined/tensorpack/_tensorpack_trial.py

```diff
@@ -466,18 +466,15 @@ def _load(self) -> None:
             backbone_weights_path = self.trial.load_backbone_weights()
             if backbone_weights_path:
                 self.load_path = pathlib.Path(backbone_weights_path)
+        else:
+            self.load_path.joinpath("checkpoint")
```

Comment:
Shouldn't this be `self.load_path = self.load_path.joinpath(...`?

Reply:
Ah yes it should